### PR TITLE
SpanBuilder deprecate startManual

### PR DESCRIPTION
**Pull request by pavolloffay**
Related to https://github.com/opentracing/opentracing-java/issues/222#issuecomment-347887141 and https://github.com/opentracing/opentracing-java/pull/219#issuecomment-348088037

### CODE REVIEW

File: opentracing-api/src/main/java/io/opentracing/Tracer.java [hunk 1]
```diff
@@ -225,6 +225,12 @@
          */
         Scope startActive(boolean finishSpanOnClose);
 
+        /**
+         * @deprecated use {@link #startManual} or {@link #startActive} instead.
```
Comment (objectiser) [inline opentracing-api/src/main/java/io/opentracing/Tracer.java]: Should be "@link #start" - and don't think this should reference startActive as is referring to the non-active behaviour.

Comment (pavolloffay) [reply opentracing-api/src/main/java/io/opentracing/Tracer.java]: oh yes, fixed

**Comment by tedsuo** [comment]
Sounds good

**Comment by carlosalberto** [comment]
Since the plan is to release rather soon the final RC for 0.31, it would be lovely to include this change in place. Any objections to have these changes merged?

**Comment by malafeev** [comment]
I think it's OK to merge as approved by several persons

**Pull request merged**